apps/web/src/pages/studio-onboarding/components/Wrapper.tsx

```diff
@@ -1,14 +1,19 @@
 import { css, cx } from '@novu/novui/css';
-import { ReactNode } from 'react';
+import { ReactNode, useEffect } from 'react';
 
 export const Wrapper = ({ children, className }: { children: ReactNode; className?: string }) => {
+  useEffect(() => {
+    document.body.setAttribute('style', 'overflow: auto');
+  }, []);
+
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** Why did we need this?‏